src/main/resources/db/changelog/db.changelog-RDM-9052.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
What is the purpose of this changelog please?

Reply:
the change log is the fix.

Reply:
I was hoping for a brief explanation of why this is needed in the context of the ticket? Especially since there are `preConditions` which it appears would not be met in other envs?

Reply:
@danlysiak I asked Henry to create a conf page documenting his findings which will explain the need for this

Reply:
The preconditions are to ensure the change log is not run when there is no need for it. i.e when the database is already there and not build from scratch.